lib/serverless-tools/cli.rb

```diff
@@ -0,0 +1,19 @@
+require "thor"
+require_relative "./comment"
+require_relative "./deployer"
+
+module ServerlessTools
+  class CLI < Thor
+    desc "comment", "create Github Issue comment body"
+    method_option :functions, :type => :string, :aliases => "-f", :default => "{}"
+    def comment
+      comment = Comment.build(options[:functions])
+      puts "::set-output name=comment::#{comment}"
+    end
+
+    desc "deploy", "publishes and deploys the specified lambda functions"
+    def deploy(action, function=nil)
+      Deployer.deploy(action: action, function: function)
+    end
```

Comment:
This seems to be different from the comment command? Could they both be the same? e.g.
```
method_option :function, :type => :string, :aliases => "-f", :default => "{}"
def deploy(action, function=nil)
      Deployer.deploy(action: action, function: options[:function])
    end
```
or does that not work for some reason?

Reply:
Correct, they're different.
TBH, this was just a personal preference/judgement call. I felt the option was more appropriate for (comment) the command.

```
serverless comment -f '{"function": "Success"}'
```
vs
```
serverless deploy build
```
I guess my feeling was that the action for the deploy command (in this case build) was an extension of the command, where the function json was an argument. I'm totally happy to just have them the same, but that was the rationale.